lms/templates/courseware/progress.html

```diff
@@ -158,7 +158,8 @@ <h3 class="hd hd-3" id="chapter_${loop.index}">${ chapter['display_name']}</h3>
                                         <%
                                         earned = section.all_total.earned
                                         total = section.all_total.possible
-                                        percentageString = "{0:.0%}".format( float(earned)/total) if earned > 0 and total > 0 else ""
+
+                                        percentageString = "{0:.0%}".format(section.percent_graded) if earned > 0 and total > 0 else ""
```

Comment:
Curious: I wonder why earned needs to be `> 0` here.  Shouldn't it be `earned >= 0`?

Reply:
@nasthagiri `section.percent_graded` is performing a `total > 0` check to see if calculating ratio is possible and returning `0.0` if it is not. And when calculating ratio is not possible we want empty string `""` displayed instead of `0.0`. For that purpose checking `total > 0` should be enough and there doesn't seem to be a need for `earned > 0` nor `>= 0`.